SUMMARY.md

```diff
@@ -23,7 +23,8 @@
 * [Node Drain](reference/node-drain.md)
 
 ## Additional Information
-* [Upgrade](reference/upgrades.md)
+* [Upgrade](reference/upgrades.md) 
+  * [Manual Upgrade](reference/manual-upgrade.md)
```

Comment:
Upgrade Support for 1.0.x instead of Manual Upgrade?

Reply:
Updated.